.env.dev.docker

```diff
@@ -9,6 +9,7 @@ REDIS_HOST=redis
 FREE_CHARGING_SERVICE=False
 FREE_CERT_INSTALL_SERVICE=True
 ALLOW_CERT_INSTALL_SERVICE=True
+USE_CPO_CERT_INSTALL_SERVICE=False
```

Comment:
Could this be more generic like `CPO_BACKEND_AVAILABLE`? Then we can use it to determine whether to mock the CSMS responses for all scenarios like this